api-1.10/src/main/java/org/openmrs/module/metadatadeploy/handler/impl/OrderFrequencyDeployHandler.java

```diff
@@ -11,23 +11,27 @@
 @OpenmrsProfile(openmrsVersion = "1.10.0")
 @Handler(supports = OrderFrequency.class)
 public class OrderFrequencyDeployHandler extends AbstractObjectDeployHandler<OrderFrequency> {
-
-    @Autowired
-    @Qualifier("orderService")
-    private OrderService orderService;
-
-    @Override
-    public OrderFrequency fetch(String uuid) {
```

Comment:
It would be better and easier to review if you do not include any formatting changes.

Reply:
I'm using OpenMRS formater on eclipse. Maybe the file had a different format?

Reply:
I'm using OpenMRS formater on eclipse. Maybe the file was formatted with a different formater?

Reply:
most likely